IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_VoltageInstrumentTransformer" UniqueId="4b47169f-04e8-f329-e589-ca2bfe243b09" ApplicableType="IfcFlowInstrument/VOLTMETER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="SecondaryVoltage_0jvGIYxLz90vG7yrY_qQ8T" />
+		<DocProperty xsi:nil="true" href="AccuracyClass_32YEer4tP9EB633g5FK49p" />
+		<DocProperty xsi:nil="true" href="AccuracyGrade_3y6bNc8F56Hexr9vdk2L" />
+		<DocProperty xsi:nil="true" href="RatedVoltage_1uw9k0qSWHuO00025QrEV" />
```

Comment:
This is on Pset_ElectricalDeviceCommon, suggestion: remove
if this refers to the instruments operating power supply voltage

Reply:
Just discussed!